modules/policy-criteria.adoc

```diff
@@ -592,7 +592,7 @@ AND, OR
 *Runtime* (when used with a Runtime criterion)
 
 | Replicas
-| The number of deployment replicas.
+| The number of deployment replicas. If you use `oc scale` to scale the deployment replicas from 0 to a number, then the admission controller will block this action if the deployment violates a policy.
```

Comment:
```suggestion
| The number of deployment replicas. If you use `oc scale` to scale the deployment replicas from 0 to a number, then the admission controller blocks this action if the deployment violates a policy.
```

Future tense is OK but at the same time it's best to avoid it in case an action does not occur as intended.